build/Targets/Packages.props

```diff
@@ -151,7 +151,7 @@
     <SystemConsoleVersion>4.3.0</SystemConsoleVersion>
     <SystemDiagnosticsContractsVersion>4.3.0</SystemDiagnosticsContractsVersion>
     <SystemDiagnosticsDebugVersion>4.3.0</SystemDiagnosticsDebugVersion>
-    <SystemDiagnosticsFileVersionInfoVersion>4.3.0</SystemDiagnosticsFileVersionInfoVersion>
+    <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
```

Comment:
FWIW, when I tried to "fix" my branch to produce usable NetCore package after recent merge from master, 4.3.0 appeared to be an issue. I had other problems as well (and so I have given up waiting for Andy to fix this ;), but I did see lots of errors like:
```
CSC : error : assembly specified in the dependencies manifest was not found -- package: 'system.diagnostics.fileversioninfo', version: '4.3.0', path: 'runtimes/win/lib/netstandard1.3/System.Diagnostics.FileVersionInfo.dll' [C:\CoreFxLab\corefxlab\src\System.Drawing.Graphics\System.Drawing.Graphics.csproj]
```